docker/Dockerfile

```diff
@@ -10,8 +10,8 @@
 ### The date in the Server command must either match, or be more recent than,
 ### the date in the FROM command
 ### Date format is ISO year/month/day
-FROM archlinux:20200505
-RUN echo "Server=https://archive.archlinux.org/repos/2020/05/05/\$repo/os/\$arch" | tee /etc/pacman.d/mirrorlist
+FROM archlinux:base
+RUN echo "Server=https://archive.archlinux.org/repos/month/\$repo/os/\$arch" | tee /etc/pacman.d/mirrorlist
```

Comment:
What's changed here? I would recommend keeping a date versioned release as otherwise, we will have the same problem in the future that caused us to have to add this. One day arch will update base and it'll break something and fixing it will be way more urgent

Reply:
Roger that. I had a stupid reason, but on reflection, you're right.

Reply:
Just to kick the horse while it is down. We should always maintain a specific date, this is mainly to enable us to keep the robots in sync with the docker images